Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -36,11 +36,19 @@ class ShopperInsightsViewController: PaymentButtonBaseViewController {
         view.textField.text = "4082321001"
         return view
     }()
+
+    lazy var sessionIDView: TextFieldWithLabel = {
```

Comment:
In my opinion we don't need a text field input for this value because a user would never be inputting this data, it would come from the merchants server. For now we can probably hard code a value and then update the demo app client to include the value from the server when it's available

Reply:
Do you think a toggle would be better? Or don't add anything to the UI and hard code into the demo app instead?

Reply:
Hmmm we could add a toggle, or also might not be necessary because the future iteration of this will remove the existing non-sessionID pattern so we will always be passing the session ID from the server (so no toggle will be needed). I think I lean just hard code it to always pass a session ID now (no toggle), but open to other's thoughts!

Reply:
ok! I thought it would be the Merchant entering in a session ID but if it comes from the server, then hard coding makes sense until server SDK updates on their end